lib/mkFlake.nix

```diff
@@ -161,7 +161,7 @@ let
                           config = selectedNixpkgs.config // config.nixpkgs.config;
                         } // { inherit (selectedNixpkgs) name input; };
                 }
-              else { _module.args.pkgs = selectedNixpkgs; })
+              else { })
```

Comment:
Is this change necessary? i feel like this would break something

Reply:
I don't remember why exactly this was necessary, I want to look into it later tonight. The tests were green though so...?